web/logout.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<!--
+    Just loading this page fires the logout.  The only reason to be directed to this page is to perform a logout.
+    Do all confirmations in the client script.
+-->
+<html>
+    <head>
+        <title>Newberry Paleography Logout</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="css/style.css" type="text/css" media="screen, projection">
+        <style>
+            #links-out{
+                
+            }
+            
+            #links-out a{
+                display: block;
+            }
+            
+            #status{
+                
+            }
+        </style>
+    </head>
+    <body>
+        <header></header>
+        <h1>Newberry Paleography Logout</h1>
+        <div id="status">
+            Logging Out...
+        </div>
+        
+        <h2>Site Links</h2>
+        <div id="links-out">
+            <a href="http://paleo.rerum.io">Newberry Paleography Login</a>
```

Comment:
We probably don't need anything French and Italian here, but I think it is not important to solve that now.